modules/sdk-coin-algo/src/algo.ts

```diff
@@ -121,6 +137,51 @@ export interface VerifiedTransactionParameters {
   numberSigners: number;
 }
 
+export interface RecoveryOptions {
+  backupKey: string;
+  userKey: string;
+  rootAddress: string;
+  recoveryDestination: string;
+  bitgoKey?: string;
+  walletPassphrase?: string;
+  fee: number;
+  firstRound?: number;
+  lease?: Uint8Array;
+  note?: Uint8Array;
+  nodeParams: NodeParams;
+}
+
+interface RecoveryInfo {
+  id: string;
+  tx: string;
+  coin: string;
+  fee: number;
+  firstRound: number;
+  lastRound: number;
+  genesisId: string;
+  genesisHash: string;
+  lease?: Uint8Array;
+  note?: Uint8Array;
```

Comment:
should be a readable string input with a max length of 1024, convert it into a Uint8Array in the method with
`new Uint8Array(Buffer.from(note, 'utf-8'))` 
```suggestion
  note?: string;
```